lib/Gateway.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-new-func */
```

Comment:
can we move these disable rules to the specific lines in question rather than the whole file where theres only a few use cases, otherwise eval's might creep in elsewhere for example

Reply:
about async functions, I really think there is no reason here. Users will only do some value parsing like transform an integer value to a rgb string and things like that, I think async will just make it more more complicated for no reason

Reply:
agree, just maybe make is clear in the gui that it must respond synchronously

Reply:
done :)